### ISSUE TITLE
Improving the layout toggle

### ISSUE THREAD

**Issue by hanneslsm** [WordPress/gutenberg#51172]
## What problem does this address?
Even though I am experienced with the Gutenberg editor, I am always getting confused by the Layout toggle, because the copy, description and order are not ideal.
<img width="274" alt="image" src="https://github.com/WordPress/gutenberg/assets/17861443/fceeed22-92ad-4eab-ac9a-0fe8434a004a">

1. `Full width → wide → content` is how we perceive layouts. So, let's use this info as a mental model.

2. "Inner blocks use content with" is very abstract. What does "_using_" mean? It implies that something is _happening_ there? But they are _set_ to a width, so they are _static_, right?
I'm suggesting something like "Constrain width of inner blocks". 
Or even better, following basic UX copy rules for toggles, something like "**Enable** inner blocks width constraining" would be possible. 
3. Actually, using the mental model, it'd be good to include the Full Width. "**Disable** full width" could work, but is also not ideal for a toggle. 
4. Next, it's confusing that the "wide" input comes after the content input. I'd suggest switching both input fields.

The current mental model is something like  `Fullscreen NO / Constrain YES → Content → Wide` which is … not ideal.

@annezazu Maybe this time a good issue for the UX & Polish board? ;). If this needs more discussion, how can these kind of issues get prioritised? This would easily increase the UX, the current implementation makes the editor so much harder to use.

**Comment by annezazu** [comment]
Thanks for opening! This matches the discussion here: https://github.com/WordPress/gutenberg/issues/42385 and here https://github.com/WordPress/gutenberg/issues/36082 Mind defaulting to those? This is far too big for the UX and Polish board as it touches on so many parts of the layout controls and that board is meant to be quickly actionable by devs.

**Comment by hanneslsm** [comment]
> This is far too big for the UX and Polish board 

Yeah, I see what you mean. However, I fear it will take a while until a decision is made in the linked issues. Until then, we could just change the copy. Again, a good practise is using "Enable" or "Disable" for toggles.

**Comment by annezazu** [comment]
I hear you. I have a ton of issues personally that I wanted to add to the board too! It's one effort to address items and we'll have more to come. Let's see what folks think around copy and how that can be iterated upon.
